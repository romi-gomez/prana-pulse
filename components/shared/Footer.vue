<template>
    <footer class="bg-mainDark text-white p-4 mt-auto">
      <div class="container mx-auto text-center">
        <p>&copy; 2023 Your Company. All rights reserved.</p>
      </div>
    </footer>
  </template>
  
  <script setup>
  </script>
  
  <style scoped>
 </style>