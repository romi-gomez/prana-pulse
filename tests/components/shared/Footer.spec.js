import { mount } from '@vue/test-utils'
import Footer from '@/components/shared/Footer.vue'

describe('Footer', () => {
  test('renders correctly', () => {
    const wrapper = mount(Footer)
    expect(wrapper.element).toMatchSnapshot()
  })

  test('contains copyright text', () => {
    const wrapper = mount(Footer)
    expect(wrapper.text()).toContain('Â© 2023 Your Company. All rights reserved.')
  })
})