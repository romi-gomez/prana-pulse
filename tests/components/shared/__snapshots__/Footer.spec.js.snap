// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  class="bg-mainDark text-white p-4 mt-auto"
>
  <div
    class="container mx-auto text-center"
  >
    <p>
      Â© 2023 Your Company. All rights reserved.
    </p>
  </div>
</footer>
`;
